app/controllers/deceased_settlor/SettlorDateOfDeathYesNoController.scala

```diff
@@ -38,13 +38,13 @@ class SettlorDateOfDeathYesNoController @Inject()(
                                          identify: IdentifierAction,
                                          getData: DraftIdRetrievalActionProvider,
                                          requireData: DataRequiredAction,
-                                         formProvider: SettlorDateOfDeathYesNoFormProvider,
+                                         yesNoFormProvider: YesNoFormProvider,
                                          requiredAnswer: RequiredAnswerActionProvider,
                                          val controllerComponents: MessagesControllerComponents,
                                          view: SettlorDateOfDeathYesNoView
                                  )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
 
-  val form: Form[Boolean] = formProvider()
+  val form: Form[Boolean] = yesNoFormProvider.withPrefix("settlorDateOfDeath")
```

Comment:
Should prefix not  be "settlorDateOfDeathYesNo"?

Reply:
Yes! Thank you. I have changed that